src/beancount.pest

```diff
@@ -1,13 +1,16 @@
 //// General primitives
 bool = { ^"true" | ^"false" }
-indent = { WHITE_SPACE+ }
-comment = { WHITE_SPACE* ~ ";" ~ (!NEWLINE ~ ANY)* }
-eol = { NEWLINE | comment ~ NEWLINE }
-key = { !":" ~ ASCII_ALPHA_LOWER ~ (ASCII_ALPHANUMERIC | DASH | "_")* }
+indent = { ws+ }
+comment = { ws* ~ ";" ~ (!NEWLINE ~ ANY)* }
+eol = { ws* ~ NEWLINE | comment ~ NEWLINE }
+indented_comment_line = { indent ~ comment ~ NEWLINE }
+key = { ASCII_ALPHA_LOWER ~ (ASCII_ALPHANUMERIC | DASH | "_")* }
 key_value_value = { quoted_str | account | date | commodity | tag | bool | amount | num_expr | str }
-key_value = { key ~ ":" ~ WHITE_SPACE? ~ key_value_value }
+key_value = { key ~ ws* ~ ":" ~ ws* ~ key_value_value }
```

Comment:
I don't think there can be whitespace before the colon?

Reply:
Correct - fixing.